cmd/rm-main.go

```diff
@@ -285,7 +351,13 @@ func mainRm(ctx *cli.Context) error {
 	// Support multiple targets.
 	for _, url := range ctx.Args() {
 		if isRecursive {
-			err = removeRecursive(url, isIncomplete, isFake, older)
+			if isHostName(url) {
+				if isNamespace {
+					err = removeSiteRecursive(url, isIncomplete, isFake, older)
```

Comment:
Unable to understand the need of this function.

Current code fails for below commands
```bash
$ mc rm --recursive --fake --force play
mc: <ERROR> Failed to remove `play` recursively. Bucket `` does not exist.
```

What I am able to visualize with `--namespace` flag is by running `mc rm --recursive --fake --force play`, it could get list of buckets from `play` alias and keep calling `removeRecursive()` with each bucket URL.